CMakeLists.txt

```diff
@@ -526,12 +526,30 @@ install (EXPORT griddyn-targets
 # CTest
 # -------------------------------------------------------------
 
-enable_testing()
-add_test(NAME testComponents COMMAND testComponents)
-
-add_test(NAME testLibrary COMMAND testLibrary)
+if(TEST_ENABLE)
+    enable_testing()
+    add_test(NAME testComponents COMMAND testComponents)
+    add_test(NAME testLibrary COMMAND testLibrary)
+    add_test(NAME testSystem COMMAND testSystem)
+    add_test(NAME testExtra COMMAND testExtra)
```

Comment:
The test extra has mainly been for some experimental runs I have done on occasion, such as timing tests and scalability testing, probably not something that should be run as part of ctest